src/test/unit/test_auto_instrument_handler.py

```diff
@@ -1,11 +1,23 @@
-import imp
+import importlib
+import sys
 import traceback
 
 import mock
 import pytest
 
 from lumigo_tracer.auto_instrument_handler import ORIGINAL_HANDLER_KEY, _handler
-from lumigo_tracer.libs.lambda_runtime_exception import FaultException
+
+if sys.version_info >= (3, 7):
```

Comment:
AFAIK, the unit tests run once during our CI: with python3.7. 
I.e. your `else` will never be executed as part of our CI, so this is basically a stale code.
IMO - you don't need it at all. 3.6 is EOF

Reply:
I wanted to deprecate support for 3.6, but it turns out that we still have active customers using lambda 3.6...

Anyways because the imported library is for worst case scenario, I feel good only using the awslambdaric version that supports 3.7+ (It unofficially supports 3.6 as specified in their setup.py file in version 2.0.8)